--- spring_0.79.0.2/rts/System/SpringApp.cpp	2009-05-23 02:59:41.000000000 +0200
+++ spring_0.79.0.2/rts/System/SpringApp.cpp.new	2009-05-26 09:19:57.000000000 +0200
@@ -590,8 +590,8 @@
 void SpringApp::LoadFonts()
 {
 	// Initialize font
-	const std::string fontFile = configHandler->GetString("FontFile", "fonts/FreeSansBold.otf");
-	const std::string smallFontFile = configHandler->GetString("SmallFontFile", "fonts/FreeSansBold.otf");
+	const std::string fontFile = configHandler->GetString("FontFile", "/usr/share/fonts/TTF/FreeSansBold.ttf");
+	const std::string smallFontFile = configHandler->GetString("SmallFontFile", "/usr/share/fonts/TTF/FreeSansBold.ttf");
 	const int fontSize = configHandler->Get("FontSize", 23);
 	const int smallFontSize = configHandler->Get("SmallFontSize", 14);
 	const int outlineWidth = configHandler->Get("FontOutlineWidth", 3);
